Forte.EpiServer.AzureSearch/ContentExtractor/ContentExtractorController.cs

```diff
@@ -0,0 +1,82 @@
+using System.Collections.Generic;
+using System.Linq;
+using EPiServer.Core;
+using Forte.EpiServer.AzureSearch.Extensions;
+using Forte.EpiServer.AzureSearch.Model;
+
+namespace Forte.EpiServer.AzureSearch.ContentExtractor
+{
+    public class ContentExtractorController : IContentExtractorController
+    {
+        public const string BlockExtractedTextFragmentsSeparator = " ";
+        private readonly IEnumerable<IContentExtractor> _extractors;
+
+        public ContentExtractorController(IEnumerable<IContentExtractor> contentExtractors)
+        {
+            _extractors = contentExtractors;
+        }
+
+        public IEnumerable<string> Extract(IContent content)
```

Comment:
It seems that implementation of those Extract(IContent content) and Extract(IContentData content) are almost identical. Can we avoid duplicated code?